### ISSUE TITLE
Exodus Path for AppRole

### ISSUE THREAD

**Issue by 7hunderbird** [genesis-community/concourse-genesis-kit#64]
Today I was examining the vault policy created by the **setup-approle** addon.

And saw this policy:

```
$ safe vault policy read genesis-pipelines
# Allow the pipelines to read all items within Vault, and write to secret/exodus (for genesis exodus data)

path "/secret/*" {
  capabilities = ["read", "list"]
}

path "/secret/*" {
  capabilities = ["create", "read", "update", "list", "delete"]
}
```

Is the second path meant to be `/secret/exodus/*` instead?

It may be a combo of how the path for [secrets_mount](https://github.com/genesis-community/genesis/blob/2174240cee2d95c787dfb46f0f517b1a41c0c549/lib/Genesis/Vault.pm#L438) gets set in genesis.

And how the [matching_mount](https://github.com/genesis-community/concourse-genesis-kit/blob/7b25d26bf96afba1c1c7eab05e4a88e7bb7321f0/hooks/addon#L268) function affects the [exo_mnt](https://github.com/genesis-community/concourse-genesis-kit/blob/7b25d26bf96afba1c1c7eab05e4a88e7bb7321f0/hooks/addon#L252) value in the addon script.

For now, the security impact is that the app-role _can_ write to anything in Vault, instead of only in the exodus path.

**Comment by 7hunderbird** [comment]
Thanks @dennisjbell for the update. 👍

**Comment by dennisjbell** [comment]
Fixed in v3.21.3
